src/bracket-ipv6-address.cpp

```diff
@@ -0,0 +1,70 @@
+// @file bracket-ipv6-address.c
+//
+// This tool takes an IP address as a string.  If the IP address is IPv4 it
```

Comment:
I'd argue that what the tool does it more important than the copyright statement.  (Why would you copy it if it **didn't** do what you wanted?)  On the other hand, I don't care enough to argue.